api/src/neo4j/queries/search/modelSearch.js

```diff
@@ -7,6 +7,49 @@ import {
   getScore,
 } from 'neo4j/queries/search/helper';
 
+const fetchIds = async ({ component, term, model, version, limit }) => {
```

Comment:
Maybe we could rename this function to something like `searchForIds` instead? This way there would be a clearer distinction between the purpose of this function compared to the purposes of other functions like as `fetchGenes`.